src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -82,7 +84,54 @@ func init() {
 	rand.Seed(time.Now().UnixNano())
 }
 
+func logPanic(proxyAddr string, msg string) {
+	cfg, err := config.Init(packageVersion)
+	if err != nil {
+		panic("Error initializing config")
+	}
+
+	// Create the client-side proxy.
+	c := &client.Client{
+		// TODO: Next port in case the regular one hasn't been closed properly
+		Addr:         proxyAddr,
+		ReadTimeout:  0, // don't timeout
+		WriteTimeout: 0,
+	}
+	ready := make(chan bool)
+	go c.ListenAndServe(func() { ready <- true })
+	<-ready
```

Comment:
Ok. I was thinking on using a different address for proxy instead of the default (this is TODO), since the port might be on use and produce an error when trying to open. However, if we are not going to use Loggly through a proxy, this should just be removed.